lib/hydra/works.rb

```diff
@@ -64,21 +64,38 @@ module Works
     autoload :GenerateThumbnail,                   'hydra/works/services/generic_file/generate/thumbnail'
     autoload :PersistDerivative,                   'hydra/works/services/generic_file/persist_derivative'
 
-    # model validations
-    def self.collection? collection
+    # model validations 
+    def self.works_collection? collection
```

Comment:
Yes, it does return no method, so I will change it to be like you propose,
and you can let me know what you think when I push the PR.

Thanks!
Jose

On Mon, Aug 3, 2015 at 3:03 PM, Esmé Cowles <notifications@github.com>
wrote:

> In lib/hydra/works.rb
> <https://github.com/projecthydra-labs/hydra-works/pull/182#discussion_r36118757>
> :
>
> > @@ -64,21 +64,38 @@ module Works
> >      autoload :GenerateThumbnail,                   'hydra/works/services/generic_file/generate/thumbnail'
> >      autoload :PersistDerivative,                   'hydra/works/services/generic_file/persist_derivative'
> >
> > -    # model validations
> > -    def self.collection? collection
> > +    # model validations
> > +    def self.works_collection? collection
>
> I would expect that test:
>
> expect( pcdm_coll.works_collection? ).to be false
>
> to raise a NoMethodError -- so maybe the more appropriate test for
> non-Hydra::Works objects would be something like:
>
> expect( pcdm_coll.respond_to? :works_collection ).to be false
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/projecthydra-labs/hydra-works/pull/182/files#r36118757>
> .
>